clang/include/clang/Basic/Cuda.h

```diff
@@ -121,6 +121,11 @@ enum class CudaArch {
   GFX1151,
   GFX1200,
   GFX1201,
+  GFX9_GENERIC,
```

Comment:
I think you might want to group them with corresponding non-generic targets such that we can directly use `>=` and `<=` to compare if a GFX version falls into one category if necessary. I'm not sure if we have this kind of use case, but it is common for the other targets. Ignore this if we don't.